go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -1145,6 +1163,20 @@ func (nc *DefaultNodeNetworkController) Start(ctx context.Context) error {
 		ovspinning.Run(nc.stopChan)
 	}()
 
+	if config.OvnKubeNode.Mode == types.NodeModeDPU {
+		zone, err := getOVNSBZone()
```

Comment:
this works for us in this particular model (ovn-ic), but this will break ovn-central altogether.
the aforementioned mode is not deprecated in upstream.

for now I am ok with the change since it suits our usecase.
please add a TODO here to fix the ovn-central behaviour. we will tackle it later on.